Makefile.common

```diff
@@ -157,10 +157,15 @@ ifeq ($(TARGET), retroarch_3ds)
    OBJ += frontend/drivers/platform_ctr.o
 endif
 
-# Git
+# Git Version
 
 GIT_VERSION := $(shell git rev-parse --short HEAD 2>/dev/null)
 ifneq ($(GIT_VERSION),)
+   LAST_GIT_VERSION := $(shell cat .last-git-version 2>/dev/null)
```

Comment:
Any reason that's not in the object directory? Remember that dotfiles aren't hidden on Windows.

Other than that, LGTM

Reply:
The reasoning is that at the time of this check the object directory might not exist.
This file should be as ephemeral as possible and I was not really aiming for hidding the file specifically.

Actually you reminded me that it should be removed in the `clean` target. Will update the PR for that.

Reply:
Personally I'd copypaste an mkdir into there. We have half a dozen copies already, another one won't break anything.

Also consider removing the $(OBJDIR)/version_git.o rule completely. It's currently identical to the generic .c rule, except its dependencies and a missing $(CPPFLAGS), both of which seem like bugs in version_git.

Reply:
Alright that makes sense!  I updated the PR now with your suggestions. Indeed it looks cleaner, thanks!
Tested and working fine on my side. Let me know if you are okay with it now.

Reply:
Yep, looks good to me. Fixes bugs and removes more than it adds, PRs don't get much better than that.